libs/frontend/feature-render-builder/src/lib/use-render-builder.ts

```diff
@@ -1,16 +1,54 @@
+/* eslint-disable vue/one-component-per-file */
 import {
   IUseRender,
   IUseRenderOptions,
   useRender,
+  getReusableStyle,
 } from '@pubstudio/frontend/feature-render'
 import { NotFound } from '@pubstudio/frontend/ui-widgets'
 import { computed, defineComponent, h } from 'vue'
 import { renderPage } from './render-builder'
+import { IRawStyleRecord, rawStyleRecordToString } from '@pubstudio/frontend/util-render'
+import {
+  activeBreakpoint,
+  descSortedBreakpoints,
+} from '@pubstudio/frontend/feature-site-source'
+import { Css } from '@pubstudio/shared/type-site'
 
 export const useRenderBuilder = (options: IUseRenderOptions): IUseRender => {
   const { site, activePage } = options
   const render = useRender(options)
 
+  const reusableStyle = computed(() => {
```

Comment:
I think the bug show up because it's meant to be flattened based on the active breakpoint, like how component custom style did, instead of being converted to media query css.
I think there's a chance that we could do this with [container queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_container_queries), which might make the code a little bit cleaner (or not 🤔?), but I'm not sure if the [browser support](https://caniuse.com/css-container-queries for that feature is good enough to be considered common.

Reply:
Got it! I like the idea of using container queries, but this works fine for now. I'm not completely sure how to reproduce the original bug, but everything seems correct in the templates. I'll keep testing!